vite.config.ts

```diff
@@ -1,6 +1,6 @@
-import { defineConfig } from 'vite';
-import react from '@vitejs/plugin-react';
-import { viteStaticCopy } from 'vite-plugin-static-copy';
+import { defineConfig } from 'vite'
```

Comment:
Why are you removing the semicolumns here

Reply:
was not done manually, according to my lens it came from a merge from vnext from a change 3 weeks ago. Added them  back